core/src/main/java/com/netflix/conductor/core/dal/ExecutionDAOFacade.java

```diff
@@ -331,20 +331,43 @@ public void removeFromPendingWorkflow(String workflowType, String workflowId) {
     /**
      * Removes the workflow from the data store.
      *
-     * @param workflowId the id of the workflow to be removed
+     * @param workflowId the workflow to be removed
```

Comment:
why change this wording? the "id" is more accurate. Your wording makes it seem like a workflow object

Reply:
You are correct, I will revert this!